documentation/Windows-Installation-Guide.md

```diff
@@ -0,0 +1,399 @@
+
+```
+psql -d tombolo -U tombolo
+Enter the password as tombolo and now Type the following commands
+CREATE EXTENSION postgis;
+SET NAMES ‘UTF8’;
+
+```
+![Pg create extension](https://user-images.githubusercontent.com/19670372/33494854-7cb9e724-d6bc-11e7-8391-0e02f9bdf069.png)
+
+- Type ```\q``` to quit
+- Now run the sql script to setup your database
+
+```
+psql -d tombolo -U tombolo < src\main\resources\sql\create_database.sql
+Enter the password as ‘tombolo’
+
+```
+![Tombolo script output](https://user-images.githubusercontent.com/19670372/33494821-79893352-d6bc-11e7-8447-7f86b0a3fd61.png)
+
+#### Test Database
+
+- The steps for setting up the test database is same as setting up the main database. Thus follow the steps 
+starting from **Main Database** section and replace **tombolo** with **tombolo_test** everywhere even 
+for the passwords. Below are just the screenshots for the steps:
+![Tombolo Test database](https://user-images.githubusercontent.com/19670372/33494852-7c82926a-d6bc-11e7-8ec8-c6299e365501.png)
+![Tombolo Test database 2](https://user-images.githubusercontent.com/19670372/33494851-7c6ad0f8-d6bc-11e7-9074-9b71f0482550.png)
+
+###### Congratulations now your both the databases are setup. 
+
+### Build and Run the project
+
+#### Build
+
+- Now you are all set to build the project **TomboloDigitalConnector**. To build the project 
+in **Command Prompt** type the following
+
+```
+gradle clean build
+```
+- Running the aforementioned command should start downloading the packages required to run **DigitalConnector**, 
+once the packages is downloaded it will start building the **DigitalConnector** and start running the tests, on a 
+machine with 8gb of ram it should not take more than 15 minutes to run the tests, once the downloads are done.
+- Below are couple screenshots of the start of the process and end of the process.
+![Gradle build](https://user-images.githubusercontent.com/19670372/33494886-7f9a2e72-d6bc-11e7-880f-1e2cd2f1d2cb.png)
+![Gradle build 2](https://user-images.githubusercontent.com/19670372/33494885-7f641d8c-d6bc-11e7-8628-e52ddd280ed7.png)
+
+###### Congratulations you have successfully built the Digital Connector. Now let’s try to run an example.
+
+#### Run
+
+- Type the following command in the **Command Prompt**
+
+```
+gradle runExport ^
+-PdataExportSpecFile=src\main\resources\executions\examples\reaggregate-traffic-count-to-la.json ^ 
+-PoutputFile=reaggregate-traffic-count-to-la_output.json
+```
+
+- Below are couple of screenshots of start and end of the process.
+![Gradle run export 1](https://user-images.githubusercontent.com/19670372/33494879-7ee0a3bc-d6bc-11e7-81f6-c005e6604e92.png)
+![Gradle run export 2](https://user-images.githubusercontent.com/19670372/33494878-7ec5c844-d6bc-11e7-92cc-0296a925ebea.png)
+
+##### View output in a file
+
+```
+start notepad reaggregate-traffic-count-to-la_output.json
+```
+
+##### View output in a Folder
+
+```
+start .
+```
+
```

Comment:
My comment is a bit silly, it probably my OCD 😄 

Multiple extra blank lines are not needed, only 1 blank line is needed at the end of a file.

Reply:
Not a problem on the PR, someone else can tidy this up in the future 😉